### PR TITLE
ItemList signals additional documentation

### LINKED ISSUE #17588: bugs in signals of ItemList (item_selected) & (item_rmb_selected)
Godot version: 2.1.4 stable & 2.1.5 beta2
OS: Windows 7 x64

**Issue description:**
1) signals of ItemList (item_selected), has an bug - then the signal does not emit if you select the same item again. Even if you changed the tabs and there you selected a new item, and when you return to the previous tab, you can not select the previously selected one, again - (the signal will not be emitted).
2) signals of ItemList (item_rmb_selected), does not work at all, under any conditions.

Steps to reproduce:
0) download, open and run the sample project
1) in the first tab of the list of items, select item # 0
2) check the inscription with the indicator (the line above the tabs) it should become == 0
3) switch to the second tab, select item # 3
4) check the inscription with the indicator (the line above the tabs) it should become == 3
5) go back to the first tab, try again to select item # 0 (this does not work, it will still be equal to # 3 from the second tab)
6) read the commentators in the script, removing the Comments from the code, you can check the rest of the signals.

Minimal reproduction project:
[https://www.dropbox.com/s/vmq1jl6nh2482bl/bug_signals_for_ItemList.zip?dl=0](https://www.dropbox.com/s/vmq1jl6nh2482bl/bug_signals_for_ItemList.zip?dl=0)

### PR DESCRIPTION
**Pull request by yvie-k**
Signals:
* item_selected
* item_rmb_select

Resolves #17588

### CODE REVIEW

File: doc/classes/ItemList.xml [hunk 1]
```diff
@@ -412,13 +412,15 @@
 				Fired when specified list item has been selected via right mouse clicking.
 				The click position is also provided to allow appropriate popup of context menus
 				at the correct location.
+				[code]allow_rmb_select[/code] must be enabled.
```
Comment (akien-mga) [inline doc/classes/ItemList.xml]: You can use `[member allow_rmb_select]` to get an hyperlink to the mentioned property.

Comment (Dragoncraft89) [reply doc/classes/ItemList.xml]: Changed

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**